adafruit_bus_device/i2c_device.py

```diff
@@ -77,6 +77,14 @@ def read_into(self, buf, **kwargs):
         """
         self.i2c.readfrom_into(self.device_address, buf, **kwargs)
 
+    def read_into(self, buf, **kwargs):
+        """
+        Warning: This method will be deprecated some time in the future.
+        Please start using readinto() instead.  readinto() is functionally
```

Comment:
Please use backticks around readinto so that sphinx makes it clickable. You can also use .. warning:: to make it called out in a box. http://www.sphinx-doc.org/en/stable/rest.html#directives

Reply:
fixed